src/content/docs/authors-analytics-manual.mdx

```diff
@@ -10,20 +12,21 @@ To search for engagement reports in a specific language:
 
 ![Image - Show steps to search by language on googla analytics](https://contribute.freecodecamp.org/images/google-analytics/search-by-language.png)
 
-1. From the top dropdown menu, select `News`.
-1. From the sidebar, click on `Reports`.
-1. From the secondary sidebar, select `Engagement`.
-1. Click on `Pages and Screens`.
-1. In the search bar, type the subpath for the desired language.
-1. From the dropdown under the search bar, choose `Page path and screen class`.
-
+<Steps>
+  1. From the top dropdown menu, select `News`. 2. From the sidebar, click on
+  `Reports`. 3. From the secondary sidebar, select `Engagement`. 4. Click on
+  `Pages and Screens`. 5. In the search bar, type the subpath for the desired
+  language. 6. From the dropdown under the search bar, choose `Page path and
+  screen class`.
+</Steps>
```

Comment:
This formatting doesn't look right.

Reply:
Thank you for pointing this out. It's been taken care of.

Reply:
Thanks one last nit, can we make them consistent? At the minute there are two versions, one with new lines between the steps and one with a newlines between the tags and the steps.

I prefer this one:

```jsx
<steps>

1. aaaaa
2. bbbbb
3. ccccc

</steps>
```

Reply:
Okay I think I've got it now.